src/Mod/Part/App/TopoShape.h

```diff
@@ -758,6 +758,21 @@ class PartExport TopoShape: public Data::ComplexGeoData
         }
     };
 
+    /** Make a ruled surface
+     *
+     * @param sources: the source shapes, each of which must contain either a
+     *                 single edge or a single wire.
+     * @param orientation:

```

Comment:
Missing docstring for orientation, and it probably should be an enum of some kind, instead of an int, right?

Reply:
Well, it is a value that comes through from the UI, so I'm not sure I can change it, but I can doc it.